testpmd/hooks/install.yml

```diff
@@ -1,8 +1,41 @@
 ---
+- name: Get all RuntimeClass
+  community.kubernetes.k8s_info:
+    api_version: node.k8s.io/v1
+    kind: RuntimeClass
+  register: rclass
+
+- name: Select High performance RuntimeClass
+  set_fact:
+    runtime_class: "{{ rclass | json_query(query) | first }}"
+  vars:
+    query: "resources[?handler=='high-performance'].metadata.name"
+  when: rclass|length > 0
 
 - name: Deploy the Example CNF applications
   include_role:
     name: "{{ deploy_dir }}/roles/example-cnf-app"
+  vars:
+    high_perf_runtime: "{{ runtime_class }}"
+
+- name: "Patch deployment to add annotations to testpmd pods"
+  community.kubernetes.k8s:
+    state: present
+    api_version: apps/v1
+    kind: Deployment
+    namespace: "{{ cnf_namespace }}"
+    name: "testpmd-app"
+    definition:
```

Comment:
Hi @capolrik you have an implicit override here because if a resource definition is provided, it will override api_version, name, kind, and all the rest, [here is the documentation about this behaviour](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html). 
Would it be better to move everything to definition, to avoid the default values that could overwrite all the rest and also to move definition to a template?

Reply:
You are absolutely right! I was trying to change the CR definition at the beginning but it did not work because of permission issue, the testpmd operator does not have the permission to request information about runtimeClass.
I wanted to keep it simple instead of changing permissions that escalates to many problems

Reply:
Nope, you did not get my comment, sorry, I was unclear. 
Actually, I proposed to do something like this
```
- name: "Patch deployment to add annotations to testpmd pods"
  community.kubernetes.k8s:
    state: present
    namespace: "{{ cnf_namespace }}"
    definition: |
      apiVersion: apps/v1
      kind: Deployment
      metadata:
        name: "testpmd-app"
      spec:
        template:
          metadata:
            annotations:
              irq-load-balancing.crio.io: "disable"
          spec:
            runtimeClassName: "{{ runtime_class }}"
  when:
    - runtime_class is defined
    - runtime_class|length
```

Reply:
I have updated the commit. Thanks :)

Reply:
In the same way we are applying the runtimeClassName to the trex resources in https://github.com/openshift-kni/example-cnf/pull/50/, couldn't we do something similar for testpmd?
Just note the following: there are some tnf tests that are considered "intrusive" because they delete resources and check if they're correctly recreated. One of them tests the removal of deployments. If the deployment is removed, when being recreated, it will lose all these annotations we're adding here. My point here is that we should definitely try to move this to the Ansible roles that manage the testpmd operator.

Reply:
@capolrik , I saw you were adding these specs already in https://github.com/openshift-kni/example-cnf/pull/50, so I suppose this task can be deleted here

Reply:
Yes, I was about to test this, it should be necessary.